### ISSUE TITLE
no JSON functions?

### ISSUE THREAD

**Issue by matrey** [chdb-io/chdb#128]
(JSON functions: https://clickhouse.com/docs/en/sql-reference/functions/json-functions)

```
SELECT isValidJSON('not a json');

Code: 48. DB::Exception: Functions JSON* are not supported: While processing isValidJSON('not a json'). (NOT_IMPLEMENTED)
```
This error seems to be coming from the DummyJSONParser under src/Common/JSONParsers/

Did `chdb` intentionally chose to remove JSON functions? Or is it a bug?

**Comment by auxten** [comment]
It's a chdb bug, will be fixed in v0.15

**Comment by lmangani** [comment]
Implemented in [v0.15.0](https://chdb.fly.dev/?user=default#U0VMRUNUICogZnJvbSBzeXN0ZW0uZm9ybWF0cyBPUkRFUiBCWSBuYW1l)
